docs/src/main/paradox/sbt.md

```diff
@@ -4,39 +4,29 @@ To get started with Akka gRPC read the [client](client.md) or [server](server.md
 
 ## Only generating the server or client side
 
-By default the plugin generates both a client and a server stub for Scala. 
-
 It can be configured to just generate either server or client like so:
 
 ```scala
-// This is the default
-import akka.grpc.gen.scaladsl.ScalaBothCodeGenerator
-akkaGrpcCodeGenerators := Seq(GeneratorAndSettings(ScalaBothCodeGenerator, akkaGrpcCodeGeneratorSettings.value))
+// This is the default - both client and server
+akkaGrpcTargetStubs := Seq(AkkaGrpc.Server, AkkaGrpc.Client)
 
 // only client
-import akka.grpc.gen.scaladsl.ScalaClientCodeGenerator
-akkaGrpcCodeGenerators := Seq(GeneratorAndSettings(ScalaClientCodeGenerator, akkaGrpcCodeGeneratorSettings.value))
+akkaGrpcTargetStubs := Seq(AkkaGrpc.Client)
 
 // only server
-import akka.grpc.gen.scaladsl.ScalaServerCodeGenerator
-akkaGrpcCodeGenerators := Seq(GeneratorAndSettings(ScalaServerCodeGenerator, akkaGrpcCodeGeneratorSettings.value))
-``` 
+akkaGrpcTargetStubs := Seq(AkkaGrpc.Server)
+```
 
-It can also ge configured to generate Java classes:
+What language to generate stubs for is also configurable:
+```
+// default is Scala only
+akkaGrpcTargetLanguages := Seq(AkkaGrpc.Scala)
 
-```scala
 // Java version of the default - both client and server code generated
-import akka.grpc.gen.javadsl.JavaBothCodeGenerator
-akkaGrpcCodeGenerators := Seq(GeneratorAndSettings(JavaBothCodeGenerator, akkaGrpcCodeGeneratorSettings.value))
-
-// only client
-import akka.grpc.gen.javadsl.JavaClientCodeGenerator
-akkaGrpcCodeGenerators := Seq(GeneratorAndSettings(JavaClientCodeGenerator, akkaGrpcCodeGeneratorSettings.value))
-
-// only server
-import akka.grpc.gen.javadsl.JavaServerCodeGenerator
-akkaGrpcCodeGenerators := Seq(GeneratorAndSettings(JavaServerCodeGenerator, akkaGrpcCodeGeneratorSettings.value))
+akkaGrpcTargetLanguages := Seq(AkkaGrpc.Java)
 
+// Both Java and Scala
+akkaGrpcTargetLanguages := Seq(AkkaGrpc.Java, AkkaGrpc.Scala)
```

Comment:
+1 What is more, do you know if it is possible to control the package name of the generated gRPC model classes from the outside? Because after turning off the `flat_package` we get ourselves in a similar situation where Play API is, where everything one cryptic package name deeper is scaladsl.

Reply:
I auto-disable it for Java anyways, (because it made the Java generator fail with an error) so I think that problem is solved. But let me verify.

Reply:
I had it wrong, I'll add a comment.